src/components/date-input/index.md.njk

```diff
@@ -85,11 +85,11 @@ Say ‘Enter [whatever it is]’. For example, ‘Enter your date of birth’.
 
 #### If the date is incomplete
 
-Highlight the day, month or year field where the information is missing. If more than one field is missing information, highlight the fields the user needs to fill in.<br>
+Highlight the day, month or year field where the information is fully or partly missing. If more than one field is missing information, highlight the fields the user needs to fill in.<br>
```

Comment:
Might be a better way to say it? I think the second sentence is understood to cover both cases though.

```suggestion
Highlight the day, month or year field where the information is missing or incomplete. If more than one field is missing information, highlight the fields the user needs to fill in.<br>
```

Reply:
@calvin-lau-sig7 Cheers for reviewing! Neat suggestion